lib/cypress-rails/finds_bin.rb

```diff
@@ -0,0 +1,11 @@
+module CypressRails
```

Comment:
the main focus of this PR in on this file

Reply:
Just filling in my own context here (90% of which is @maestromac's work), `FindsBin` had always been the underlying problem in trying to get to the knapsack enabled cypress program. Nick's initial suggestion to the upstream was to add the binary path as a configurable parameter, but it was seem as undesirable, the fork had changed this callable to do what we're doing here.